wiki/tiddlers/FAQ.tid

```diff
@@ -2,47 +2,46 @@ title: FAQ
 tags: odd
 
 !! I love this! Can I use for production/very important things?
-I would love to answer yes, but for now please don't do it. Noteself is still on Beta stage, and has to be extensively tested.
-But, if you are an early adopter and want to try new and exciting technologies NoteSelf is stable enough to give it a try.
+We would love to answer yes, but for now please don't do it. Noteself is still in Beta stage and has to be extensively tested.
+But if you are an early adopter and want to try new and exciting technologies NoteSelf is stable enough to give a try.
 
 !!Where is my data stored?
-Your data is stored inside your browser. We use PouchDB in the background,
-which creates a database on your browser's LocalStorage.
+Your data is stored inside your browser. We use PouchDB in the background, which creates a database in your browser's LocalStorage.
 If LocalStorage is not available it will fallback to WebSQL.
-In any case your information is stored locally within your browser.
+In any case, your information is stored locally within your browser.
 
 !!How can I sync multiple devices?
-The best way at this moment is to use a remote `couchdb` server. There is a plenty list of compatible servers, but for simplicity, price and availability we recommend [[cloudant|https://cloudant.com/]]
+The best way at this moment is to use a remote `couchdb` server. There are plenty of compatible servers but for simplicity, price and availability we recommend [[cloudant|https://cloudant.com/]]
 
 !!I just opened my Wiki in a different browser and I have nothing!
 That's because data is stored in your browser. This means inside your browser
 literally. If you open your wiki again with the browser you were previously using
 all your data should be there.
 
-!! How can I install regular tiddliwiki plugins?
-Since version `v0.14.5` is possible to install plugins in an easier way. Just go to the control panel, open the plugins tab and then go to the plugin library.
-After a plugin installation all you have to do is refresh the page. There is no need to download the file or save the wiki anymore.
+!! How can I install regular TiddlyWiki plugins?
+It is easy! Just go to the `Control Panel`, open the plugins tab and open the plugin library.
+After a plugin is installed all you have to do is refresh the page. No need to download the file or save the wiki.
 
 If you want it //"hardcoded"// into your file, install the plugin and ''after that'' open the `Control Panel` ( ,,{{$:/core/images/options-button}},, ),
  navigate to the `TiddlyPouch` tab and click on `Save config`.
 
 !! I'm a regular TW user. What are the advantages of NoteSelf?
-* Multiple devices synchronization
+* Multiple device synchronization
```

Comment:
To edit a single file you don't need to go to your fork. You can locate the file on my repo, click the edit button and GitHub will create a new branch automatically for you and allow you to make a pull request directly from that branch. Isn't that the flow you are following?